app/Models/Author.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Author extends Model
+{
+    use HasFactory;
+
+    protected $table = 'authors';
+    protected $primaryKey = 'id';
+
+    public function publisher()
+    {
+        return $this->belongsTo(Publisher::class);
+    }
+    public function book()
+    {
+        return $this->hasMany('App\Models\Book', 'book_id', 'id');
```

Comment:
```suggestion
        return $this->hasMany(Book:class);
```
Nếu đã xài tên column foreign_key đặt dạng chuẩn rồi thì k cần khai báo thêm 2 tham số phía sau đâu Laravel tự hiểu